tests/pass/shims/x86/intrinsics-x86-avx512.rs

```diff
@@ -411,6 +413,98 @@ unsafe fn test_avx512ternarylogic() {
     test_mm_ternarylogic_epi32();
 }
 
+#[target_feature(enable = "avx512vnni")]
+unsafe fn test_avx512vnni() {
+    #[target_feature(enable = "avx512vnni")]
+    unsafe fn test_mm512_dpbusd_epi32() {
+        // These inputs aim to hit overflow cases and test that A is interpreted as unsigned and B
```

Comment:
How do they do that? Which of the constants below conspire how to make overflows happen?